tensorflow/compiler/mlir/tools/kernel_gen/kernel_creator.cc

```diff
@@ -198,14 +198,15 @@ Status LowerTFToJITInvocation(mlir::ModuleOp module,
                               llvm::ArrayRef<int64_t> tile_sizes,
                               llvm::ArrayRef<int64_t> unroll_factors,
                               int64_t max_supported_rank, bool enable_ftz,
-                              bool cpu_codegen) {
+                              bool index_64bit, bool cpu_codegen,
```

Comment:
`index_64bit` is not passed on to the `TFToJITInvocationPass` here.

Reply:
Passed.